docs/installation.md

```diff
@@ -0,0 +1,59 @@
+## Installation
+
+Following ways are supported to run Kraken:
+
+- Standalone python program through Git
+- Containerized version using either Podman or Docker as the runtime
+- Kubernetes or OpenShift deployment
+
```

Comment:
Do we want to recommend running standalone to insulate the chaos tool from failures it is causing?   Not sure it is a big deal.

Reply:
Nice suggestion @mffiedler, added the recommendation.